### DISCUSSION TITLE
How many blocks does the node keeps?

### DISCUSSION THREAD

**Discussion by andreclaro** [ava-labs/avalanchego#3149] [Q&A]
Hello,

How many blocks does the node keeps when using  "pruning-enabled": true?

I have a use case where we want just to keep one month worth of data in historical data. It seems that avalanche-go does not support custom pruning settings.

Which setting(s) do you recommend to use?

Thanks!

**Comment by qedgardo** [comment]
Same question, how many blocks are kept by default?
